index.js

```diff
@@ -1 +1,104 @@
 // implement your API here
+
+const express = require('express');
+const db = require('../webapi-i-challenge/data/db');
+
+const server = express();
+
+server.use(express.json());
+// server.use(cors());
+
+server.get('/', (req, res) => {
+  db.find()
+    .then(users => {
+      res.status(200).json(users);
+    })
+    .catch(err => {
+      res.json(err);
+    });
+});
+server.get('/api/users/:id', (req, res) => {
+  const { id } = req.params;
+  db.findById(id)
+    .then(id => {
+      if (id) {
+        res.status(200).json(id);
+      } else {
+        res.status(404).json({ message: 'user does not exist' });
+      }
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ message: 'The users informaton can not be retrived ' });
+    });
+});
+
+server.post('/api/users', (req, res) => {
+  const { name, bio } = req.body;
+  if (!name || !bio) {
```

Comment:
Great use of an if statement but try and change it to be && instead of || so he user can't leave either input empty